framework/src/modules/chain/chain.js

```diff
@@ -123,9 +123,9 @@ module.exports = class Chain {
 
 			// TODO: Use the channel application state for update node info
 			// Publish an event whenever the system headers are updated.
-			this.system.on('update', nodeInfo => {
-				this.channel.publish('chain:system:updateNodeInfo', nodeInfo);
-			});
+			// this.system.on('update', nodeInfo => {
```

Comment:
We should merge https://github.com/LiskHQ/lisk/pull/3261 first and rebase this one to avoid misunderstandings.